### PR TITLE
update cibuildwheel to build bin wheel for python 3.9

### PR DESCRIPTION
**Pull request by xyb**

### CODE REVIEW

**Comment by xyb** [comment]
Travis is very slow. Do we plan to migrate to another platform, such as Github actions?

File: .travis.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ install:
   - gcc --version
   - |
     if [ -n "${XXHASH_BUILDWHEELS:-}" ]; then
-      pip install cibuildwheel
+      pip install cibuildwheel==1.7.0
```
Comment (ifduyue) [inline .travis.yml:80]: I think in some tests pip is actually from python 2, we'd like to make sure that it is pip3.
Maybe we can try `python3 -m pip install cibuildwheel`

**Comment by ifduyue** [comment]
> Travis is very slow. Have we ever thought of migrating to another platform, such as Github actions?

I'm not against this

File: .travis.yml (hunk 1 above)
Comment (xyb) [reply .travis.yml]: Sure, updated.

**Comment by ifduyue** [comment]
Merged, thanks!

**Pull request merged**